2023-03_sprint/schedule_table.part.md

```diff
@@ -0,0 +1,33 @@
+<meta name="twitter:card" content="summary_large_image">
+<meta name="twitter:title" content="PyMC - Data Umbrella: Open Source Working Sessions">
+<meta name="twitter:description" content="Beginner friendly working sessions to get started contributing to PyMC">
+<meta name="twitter:image" content="https://raw.githubusercontent.com/pymc-devs/pymc-data-umbrella/main/_static/banner_2023_03/2023_03_banner_1280×640px.png">
+
+## Schedule
+
+:::{important}
+To take part in the upcoming Open Source Working Sessions you should:
+
+1. Fill the registration form: {bdg-link-primary-line}`https://tinyurl.com/du-pymc-os-ws`
+2. RSVP on MeetUp for the events you will be attending (links on table below)
+
+:::
+
+| Event                                     | Date               | Time                    | Duration |  RSVP       |
+|-------------------------------------------|--------------------|-------------------------|----------|-------------|
+| Open Source Working Session #1            | Jul 9 (Saturday)   |  9 ET / 13 UTC / 16 EAT | 3 hours  | [MeetUp](https://www.meetup.com/data-umbrella/events/286552452/) |
```

Comment:
Here, we could have the times in this format: Mex, Col, Peru: 8 am/ Vzla, Bo: 9 am/ Arg, Chile, Brzl: 10 am. Same for the banner.